roles/tm-deepsec-aws-setup/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
Can you please either update the README.md or remove the file.

Reply:
@goneri I am planning to update, once I am able to manage the complete silent setup. As currently the last few step for silent setup is pending and need your help to resolve it :)

Reply:
It would also be great if you can explain how to manually test the role. Which AMI do you use, how you call the role, if you need some extra steps. etc.

Reply:
+1

Reply:
+1